gear/gtest/spec/test_curve_amm.yaml

```
@@ -0,0 +1,243 @@
+title: curve_amm_test
+
+programs:
+  - id: 1
+    path: fungible_token.wasm
+    init_message:
+      kind: custom
+      value:
+        name: USDC Stable Coin
+        symbol: USDC
+  - id: 2
+    path: fungible_token.wasm
+    init_message:
+      kind: custom
+      value:
+        name: USDT Stable Coin
+        symbol: USDT
+  - id: 3
+    path: fungible_token.wasm
+    init_message:
+      kind: custom
+      value:
+        name: LP Token
+        symbol: LPT
+  - id: 4
+    path: curve_amm.wasm
+    init_message:
+      kind: custom
+      value:
+          owner: 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+          token_accounts: "{1},{2},{3}"
+          amplification_coefficient : 1000
+          fee: 2
+
+fixtures:
+  - title: curve_amm
+    messages:
+      # add 10000 USDC to account 1 
+      - destination: 1
+        payload:
```

Repeating sections may be addressed using anchors.
```suggestion
        payload: &account-1
```